spec/rails_helper.rb

```diff
@@ -31,6 +31,7 @@
 
   config.before do
     mock_github
+    WebMock.stub_request(:any, /api.data.gov/).to_rack(FakeSamApi)
```

Comment:
yes! @adelevie and I worked on this approach when I was in DC and now we have a nice setup for stubbing out the SAM API